docs/sphinx/about/index.txt

```diff
@@ -59,9 +59,42 @@ than being updated whenever a new version of :omerodoc:`OMERO <>` is released.
 We expect this to result in more frequent releases to get fixes out to the
 community faster.
 
-The version number is three numbers separated by dots e.g. 4.0.0. See the
-:doc:`version history <whats-new>` for a list of major changes in each
-release.
+See the :doc:`version history <whats-new>` for a list of major changes in
+each release.
+
+.. _versioning-policy:
+
+Versioning policy
+^^^^^^^^^^^^^^^^^
+
+Bio-Formats does not currently conform to `strict semantic versioning <http://semver.org>`_.
+The following set of rules describe the current policy (using
+`RFC 2119 <https://www.ietf.org/rfc/rfc2119.txt>`_):
+
+- The version number MUST take the form X.Y.Z where X, Y, and Z are
+  non-negative integers, and MUST NOT contain leading zeroes.
+- The patch version Z (x.y.Z | x > 0) MUST be incremented if only backwards
+  compatible bug fixes are introduced. A bug fix is defined as an internal
+  change that fixes incorrect behavior.
+- The minor version Y MUST be increased when backwards incompatible changes
+  (model breaking API) are introduced to the public API. It MAY include patch
```

Comment:
Unfortunately we don't define the major version anywhere. (These issues are partly coming up because the text is very much of a "read me pedantically, this is a formal definition" style.) Could refer to, say, "major version X (from X.y.z)" or somesuch I guess?